content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -60,7 +60,7 @@ detail:
 3. A container image of the operator code.
 4. Controller code that queries the control plane to find out what SampleDB
    resources are configured.
-5. The core of the Operator is code to tell the API server how to make
```

Comment:
Imagine that this is short for: “At the core of the Operator is code. That code is there to tell the API server how to make&hellip;”

The original text makes more sense @niaomingjian so I could not accept this as-is. If you'd like to propose a different working, please feel free. It will help reviewers if you summarize the intent of that change in the PR description.

Reply:
(By the way, operators can and often do incorporate more than one control loop)